File: C/ABC006/ABC006_A_v01.c
// AtCoder Beginner Contest 006 A
// [URL]      https://atcoder.jp/contests/abc006/tasks/abc006_1
// [compiler] C (GCC 5.4.1)
// 
#include <stdio.h>

int main (void)
{
	/* variables for main process */
	int N;

	// STEP.01
	// read the target number
	scanf( "%d", &N );

	// STEP.02
	// output wheter `N` is the multiple of `3`
	if( N % 3 == 0 ) {
		printf( "%s\n", "YES" );
	} else {
		printf( "%s\n", "NO" );
	}

	// STEP.TRUE_END
	return 0;
}

File: JuliaLang(0.5.0)/ABC006/ABC006_A_v01.jl
# [contests]   ABC 006
# [task]       A  
# [URL]        https://atcoder.jp/contests/abc006/tasks/abc006_1
# [compiler]   Julia (0.5.0)
# [submission] https://atcoder.jp/contests/abc006/submissions/4381280 : AC

#===================================================================================#

function get_data_from_stdin()
	return Base.parse( Base.readline( Base.STDIN ) )
end

function abc006_a()

	# STEP.01
	# get the target number
	n = get_data_from_stdin()

	# STEP.02
	# output the result
	if Base.isequal( Base.rem(n, 3), 0 )
		Base.println( Base.STDOUT, "YES" )
	else
		Base.println( Base.STDOUT, "NO" )
	end

	# STEP.TRUE_END
	return Base.nothing

end

#===================================================================================#

Main.abc006_a()
